slackscot.go

```diff
@@ -38,18 +38,21 @@ type Slackscot struct {
 	plugins                 []*Plugin
 	triggeringMsgToResponse *lru.ARCCache
 
-	// Caching self identity used during message processing/filtering
-	selfID         string
-	selfBotID      string
-	selfName       string
-	selfUserPrefix string
-
 	// Runtime configuration options
 	namespaceCommands bool
 
 	// Slack options to apply on Run()
 	slackOpts []slack.Option
 
+	// Command identification
+	cmdMatcher CommandMatcher
+
+	// Bot matching for messages from us
+	botMatcher SelfMatcher
+
+	// Default for cmdMatcher and botMatcher
+	selfMatcher *selfIdentity
```

Comment:
I'd call this `selfIdentity` and make this a `selfIdentity` value instead of pointer.

Reply:
Fixed in ac4da90d8f954e8d161c3a57a1243eb7bd7f8095